independent-publisher-connectors/WebEx/apiDefinition.swagger.json

```diff
@@ -0,0 +1,291 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Webex Integration",
+    "description": "WebEx API for Power Platform\n\n- Setup meeting from Power Platform",
+    "contact": {
+      "name": "UCL",
+      "email": "oscar.hui.21@ucl.ac.uk"
+    }
+  },
+  "host": "webexapis.com",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/meetings": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@ocarhui empty schema is not allowed. Please remove or define a proper schema.
Please fix all empty schemas in your connector swagger file.